game_calls.cpp

```diff
@@ -37,14 +48,15 @@ std::vector<int> GetGuess()
 	return result;
 }
 
-bool CheckGuess(std::vector<int> guess, std::vector<int> code)
+bool CheckGuess(const std::vector<int>& guess, const std::vector<int>& code)
 {
 	if (guess.empty() || code.empty())
 	{
 		std::cout << "Error: Guess or Code was empty!\n";
 		return false;
 	}
 
+	// A for loop seems to make sense here; not sure why `while` is better?
```

Comment:
Yes I agree. Was initially going to use a for loop but then got a bit confused with the size_t and decided to use a while loop and didn't bother changing back.

Reply:
```
for (size_t i = 0; i < guess.size() && i < code.size(); ++i)
```